### ISSUE TITLE
Error while trying to install

### ISSUE THREAD

**Issue by ellev** [kdietrich/homebridge-devolo#31]
Getting this error can someone please help me?

pi@raspberrypi:~ $ sudo npm install -g homebridge-devolo
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git clone --depth=1 -q -b master https://github.com/kdietrich/node-devolo.git /root/.npm/_cacache/tmp/git-clone-f1e4097d
npm ERR! fatal: could not create leading directories of '/root/.npm/_cacache/tmp/git-clone-f1e4097d': Permission denied
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-02-23T22_50_37_572Z-debug.log

**Comment by kdietrich** [comment]
It looks like there is something messed up with the permissions of NPM. Could you try the steps described here: https://docs.npmjs.com/getting-started/fixing-npm-permissions ?

**Comment by ellev** [comment]
it was already a fresh install of the entire pi. Have tried the solution you proposed but still same error. No problem with other plugins.

**Comment by nicoh88** [comment]
Try this:

```
sudo su -
chown -R root /root/.npm
npm install -g homebridge-devolo —unsafe-perm
```

**Comment by ellev** [comment]
nothing worked. I tried everything you guys said and I decided to just try it one more time and all of a sudden it installed. Thanks for your help guys. I do have one more problem right now. I have 5 wall plugs and it is only showing one in the home app.

**Comment by kdietrich** [comment]
Do you mean the Devolo Smart Metering Plug (https://images-na.ssl-images-amazon.com/images/I/31iZuWFg5xL._SX355_.jpg)? Could you provide me a debug log by running `homebridge -D > logfile.txt 2>&1` and attaching the log file?

**Comment by ellev** [comment]
yes the Smart Metering Plug. I have 5 of them and it only shows one. The name is "Telenet 2". The other accessory "Inkom" is a door/window contact. 
[logfile.txt](https://github.com/kdietrich/homebridge-devolo/files/1760396/logfile.txt)

**Comment by kdietrich** [comment]
Strange, devolo-homebridge is picking up only two of your devices. Just to confirm: All five devices are included into the same Devolo Central Unit? Can I reach out to you by mail?

**Comment by ellev** [comment]
indeed very strange. Although the first time I started Homebridge with the plugin they where all in one group on mydevolo because we always switch them together. (We use them for hidden led lights and stuff in the walls of our store)
Mail: kevindl@autosoft.be

**Comment by ellev** [comment]
Omg
You are amazing!!!
 
I fixed it thanks to your debug option. The items that where not shown where not in a group “Lommel”. So they didn’t got recognized in your plugin. I went to mydevolo and put them all in the same group and it worked like a charm!!
 
Logfile is attached for your info.
 
Thanks again for your amazing support!!!!
[logfile.txt](https://github.com/kdietrich/homebridge-devolo/files/1764798/logfile.txt)
